wolox-books/src/views/BookDetail.vue

```diff
@@ -0,0 +1,101 @@
+<template lang="pug">
+  .main-container
+    navbar
+    .book-container.row
+      img.book-img(:src='book.image_url' alt='Book image')
+      .book-data.column
+        .book-title
+          h1.title
+            | {{ book.title }}
+          span.genre
+            | ({{ book.genre }})
+        .book-property
+          h2.property-name
+            | Book author:
+          h3.property-value
+            | {{ book.author }}
+        .book-property
+          h2.property-name
+            | Publisher:
+          h3.property-value
+            | {{ book.publisher }}
+        .book-property
+          h2.property-name
+            | Year of publication:
+          h3.property-value
+            | {{ book.year }}
+
+</template>
+
+<script>
+import Navbar from '../components/NavBar'
+import { getBook } from '@/services/booksService'
+
+export default {
+  data () {
+    return {
+      book: ''
+    }
+  },
+  components: { Navbar },
+  async mounted () {
+    const response = await getBook(this.$route.params.id)
+    if (response.ok) {
+      this.book = response.data
+    }
+    console.log(this.book)
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../scss/variables/colors';
+
+.book-container {
+  box-shadow: 4px 4px 10px 0 rgba(0, 0, 0, 0.5);
+  margin: 40px auto;
+  width: 70%;
```

Comment:
Don't use numbers that aren't so easy to guess. Can't you use something else? Ask me and I'll help you.